distributions/distribution-resources/src/main/resources/conf/things/readme.txt

```diff
@@ -0,0 +1,7 @@
+
+openHAB2 conf/things folder
+
+Your thing definitions go here.
+All thing files have to have the ".things" file extension and must follow a special syntax.
+
+Check out the openHAB2 documentation for more details: http://docs.openhab.org
```

Comment:
This is currently described at https://www.eclipse.org/smarthome/documentation/features/dsl.html#thing-configuration-dsl.
We either copy the content or we should as well try to automatically include it in the docs - copying might be the easier solution for the moment...